Policies/Kubernetes/container-allowed-capabilities/azurepolicy.json

```diff
@@ -0,0 +1,73 @@
+{
+  "properties": {
+    "displayName": "Ensure only allowed capabilities are used in Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy ensures only given capabilities are used inside a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.",
```

Comment:
This policy ensures only specified capabilities are defined on containers in a Kubernetes cluster.

Reply:
I've changed to `This policy ensures specified security capabilities are defined and removed inside a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.`, Happy to change it to this here, however I feel this description misses the functionality of the required drop capabilities

Reply:
I would mention containers explicitly somewhere in description as capabilities are defined on containers.